Jenkinsfile

```diff
@@ -72,7 +72,8 @@ def withPath(String path, Closure closure) {
 
 node {
     //Only run validation master/stable
-    if (env.BRANCH_NAME in ['master', 'stable']) {
+    // TODO: don't merge this change
+    if (env.BRANCH_NAME in ['master', 'stable'] || true) {
```

Comment:
I am conflicted now. I want to merge but this comment tells me not to.

Reply:
more-recent past felix has corrected less-recent past felix's mistake